package.json

```diff
@@ -56,12 +60,15 @@
     "@babel/plugin-transform-runtime": "7.12.1",
     "@babel/preset-env": "7.12.1",
     "@babel/preset-react": "7.12.5",
+    "@cypress/code-coverage": "^3.12.9",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "1.3.0",
     "@redhat-cloud-services/frontend-components-config": "^6.2.9",
     "@redhat-cloud-services/frontend-components-testing": "0.1.1",
     "@redhat-cloud-services/tsc-transform-imports": "^1.0.8",
     "@redhat-cloud-services/integrations-client": "^3.2.0",
     "@redhat-cloud-services/notifications-client": "^2.5.3",
+    "@scalprum/react-core": "^0.9.3",
+    "@simonsmith/cypress-image-snapshot": "^8.1.2",
```

Comment:
I don't think you need the visual regressions testing packages and cypress config for your tests.